tests/integration/components/mutually_exclusive/test_percentage_single_checkbox_override.py

```diff
@@ -0,0 +1,47 @@
+from tests.integration.components.mutually_exclusive.schema_urls import MUTUALLY_EXCLUSIVE_PERCENTAGE
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestPercentageSingleCheckboxOverride(IntegrationTestCase):
+    """
+    Tests to ensure that the server-side validation for mutually exclusive percentage
+    with single checkbox override function as expected. These tests emulate the non-JS version.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey('test', 'mutually_exclusive_single_checkbox_override')
+        self.get(MUTUALLY_EXCLUSIVE_PERCENTAGE)
+
+    def test_non_exclusive_answer(self):
+        # When
+        self.post({'percentage-answer': '99'})
+
+        # Then
+        self.assertInUrl('section-summary')
+        self.assertInPage('99%')
+
+    def test_exclusive_answer(self):
+        # When
+        self.post({'percentage-answer': '',
+                   'percentage-exclusive-answer': ['I prefer not to say']})
+
+        # Then
+        # self.assertInUrl('section-summary')
```

Comment:
Can this be removed?

Reply:
Whoops, they are needed, commented out when I was debugging but forgot to change back after. Will sort. 👍